code/modules/mob/living/carbon/say.dm

```diff
@@ -15,6 +15,9 @@
 	return message
 
 /mob/living/carbon/can_speak_vocal(message)
-	if(silent)
+	if(losebreath)
+		emote("gasp")
+		return 0
+	if(silent || health < 0)
```

Comment:
`if(oxyloss > 10)`

No need for the health < 0 thing or any amount of losebreath causing silence--adds  hair bit more counterplay.